examples/Input.tsx

```diff
@@ -60,6 +60,10 @@ export function Input() {
         value={inputValue}
         placeholder="MM/dd/yyyy"
         onChange={handleInputChange}
+        onKeyDown={(e) => {
+          // Fixes https://github.com/gpbl/react-day-picker/issues/2724 causing search appearing when typing '/'
+          e.stopPropagation();
```

Comment:
@gpbl 

What do you think of calling `stopPropagation` only when the key typed is `/`?

This way other use cases like `cmd/ctrl + k` are not blocked.

Reply:
Oh well why should someone call that keyboard shortcut while typing into the field? I think we'd push it too far :)

Reply:
I agree, I chose the laziest path :) It seemed unnecessary to spend more time on it. I didn't even know about the shortcut itself. It looks like they addressed the bug in some release: not sure when the fix will land on Docusaurus.

> someone tries to use the shortcut while the focus is in the input

At this point, I really hope they report a bug!